BorrowBookControl.java

```diff
@@ -2,116 +2,111 @@
 import java.util.List;
 
 public class BorrowBookControl {
-	
-	private BorrowBookUI UI;
-	
-	private library LIBRARY;
-	private member M;
-	private enum CONTROL_STATE { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
-	private CONTROL_STATE State;
-	
-	private List<book> PENDING;
-	private List<loan> COMPLETED;
-	private book BOOK;
-	
-	
+
+	private BorrowBookUI borrowBookUi;
+	private library library1;
+	private member member;
+
+	private enum BookControlState {
+		INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED
+	};
+
+	private BookControlState borrowBookState;
+
+	private List<book> pendingList;

```

Comment:
@GoJoeyGo can you change private List<book> and private list<loan> into private List<Book> and <Loan>

Reply:
what do you mean

Reply:
@GoJoeyGo <book> into <Book> and <loan> into <Loan>

Reply:
<Book> into <book>

Reply:
book into <Book> please